package.json

```diff
@@ -15,20 +15,19 @@
     "@chunkd/source-memory": "^10.1.0",
     "@linzjs/style": "^5.1.0",
     "@types/node": "^20.12.10",
-    "@types/prettier": "^3.0.0",
     "stac-ts": "^1.0.0"
   },
   "scripts": {
     "build": "tsc",
     "lint": "npx eslint . --ignore-path .gitignore",
-    "test": "node --test build/"
+    "test": "node --test"
```

Comment:
I think it would be good to limit this to the build directory, `node --test "build/**/*.test.js"` otherwise its going to scan every single file in the git repo looking

Reply:
The logic has changed, so I've changed it back to the original. A quick `time npm test` check didn't seem to show a significantly longer runtime.